docs/projects/blank/tutorial/2/level.md

```diff
@@ -0,0 +1,281 @@
+                flow = new ReceptionistFlow(CreateNewWorkerId(WorkerType),
+                    new CommandLineConnectionFlowInitializer());
+                connectionParameters = CreateConnectionParameters(WorkerType,
+                    new CommandLineConnectionParameterInitializer());
+            }
+
+            var builder = new SpatialOSConnectionHandlerBuilder()
+                .SetConnectionFlow(flow)
+                .SetConnectionParameters(connectionParameters);
+
+            await Connect(builder, new ForwardingDispatcher()).ConfigureAwait(false);
+        }
+
+        protected override void HandleWorkerConnectionEstablished()
+        {
+            Worker.World.GetOrCreateSystem<MetricSendSystem>();
+            PlayerLifecycleHelper.AddServerSystems(Worker.World);
+
+            if (level == null)
+            {
+                return;
+            }
+
+            levelInstance = Instantiate(level, transform.position, transform.rotation);
+        }
+
+        public override void Dispose()
+        {
+            if (levelInstance != null)
+            {
+                Destroy(levelInstance);
+            }
+
+            base.Dispose();
+        }
+    }
+}
+```
+
+<%(/Expandable)%>
+
+## Create a Level prefab
+
+Create a `Common` folder within the `Assets/Resources/Prefabs` directory of your Unity project.
+
+<pre>
+
+    gdk-for-unity-blank-project/workers/unity
+        ├── Assets/
+            ├── Resources/
+                ├── Prefabs/
+                    ├── <b><font color="white">Common/</font></b>
+        ...
+
+</pre>
+
+Right-click in your scene and select **3D Object** > **Plane** to create a plane with scale (10, 1, 10) whose position is (0, 0, 0). Drag it into the `Assets/Resources/Prefabs/Common` folder. This will be our level, so rename it to “Level”.
+
+<img src="{{assetRoot}}assets/blank/tutorial/2/level-prefab-inspector.png" style="margin: 0 auto; width: 100%; display: block;" />
+
+Once the Level prefab is created, you can delete the plane from the current scene.
+
+## Update worker prefabs
+
+Now that the Level prefab exists, you need to update the worker prefabs to reference it. You can find these prefabs in the **Resources** > **Prefabs** > **Worker** folder.
+
+For both `ClientWorker` and `GameLogicWorker` prefabs, click the Level field on the worker connector and select the Level GameObject you just created.
+
+Let’s test this out!
+
+If you don’t have a local deployment running, hit `Ctrl+L`/`Cmd+L` inside the Unity Editor to start one up. When it’s ready, hit play in the Unity Editor. You should observe that there’s not one - but **two** levels side-by-side!
+
+<img src="{{assetRoot}}assets/blank/tutorial/2/levels-together.png" style="margin: 0 auto; width: 50%; display: block;" />
+
+Since both worker connectors were updated to instantiate a level - both of these worker types are in the development scenes - you can expect two levels to be created when you hit the play button.
```

Comment:
```suggestion
Since both worker connectors were updated to instantiate a level—both of these worker types are in the development scenes—you can expect two levels to be created when you hit the play button.
```

Think you want am em dash here, not a hypen.